test/unit/auth.js

```diff
@@ -39,15 +40,20 @@ describe('auth', () => {
     });
 
     it('updates user profile for existing user', done => {
-      // NOTE: How do we know that this is a  previous user that has been updated?
-      const userData = users[0];
-      userData.epost = 'ukjent@ukjentsen.com';
-      auth.setOrUpdateUserData(0, userData).then(user => {
-        assert.equal(user._id, userData.sherpa_id);
-        assert.equal(user.navn, `${userData.fornavn} ${userData.etternavn}`);
-        assert.equal(user.epost, userData.epost);
-        done();
-      });
+      // Set userData to the fixture user matching test case user id
+      const userData = users[1];
```

Comment:
Personlig foretrekker jeg en blank linke etter variable deklarasjoner og påfølgende logikk for å bryte det litt opp.